DQM/TrackerMonitorTrack/src/MonitorTrackResiduals.cc

```diff
@@ -38,6 +38,11 @@ void MonitorTrackResidualsBase<pixel_or_strip>::bookHistograms(DQMStore::IBooker
     m_cacheID_ = cacheID;
     this->createMEs( ibooker , iSetup);
   }
+  std::string topFolderName_ = "SiStrip";
+  SiStripFolderOrganizer folder_organizer;
+  folder_organizer.setSiStripFolderName(topFolderName_);
+  tkhisto_ResidualsMean = new TkHistoMap(ibooker , topFolderName_ ,"TkHMap_ResidualsMean", 0.0,true);
```

Comment:
HI, I cannot find the end of job where I can delete the object. I have tried to look for some examples in cmssw and I cannot find a place where these tkhisto maps are deleted (see for example https://github.com/aspiezia/cmssw/blob/a0a714e6c65870a27ee5a346d1d0e75367573276/DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc#L153). Do you have a suggestion?